devito/interfaces.py

```diff
@@ -302,7 +302,23 @@ def _indices(cls, **kwargs):
 
     def _allocate_memory(self):
         """Allocate memory in terms of numpy ndarrays."""
+        from devito.parameters import configuration
```

Comment:
Should this import be at the top of the file? This is called often, so no point keeping it in the function.The import below is fine though.